src/filters/debug.rs

```diff
@@ -16,51 +16,43 @@
 
 use std::convert::TryFrom;
 
-use serde::{Deserialize, Serialize};
-use slog::{info, o, Logger};
-
 use crate::filters::prelude::*;
+use serde::{Deserialize, Serialize};
+use tracing::{info, instrument};
 
 crate::include_proto!("quilkin.extensions.filters.debug.v1alpha1");
 use self::quilkin::extensions::filters::debug::v1alpha1::Debug as ProtoDebug;
 
 /// Debug logs all incoming and outgoing packets
 #[derive(Debug)]
-struct Debug {
-    log: Logger,
-}
+struct Debug {}
 
 pub const NAME: &str = "quilkin.extensions.filters.debug.v1alpha1.Debug";
 
 /// Creates a new factory for generating debug filters.
-pub fn factory(base: &Logger) -> DynFilterFactory {
-    Box::from(DebugFactory::new(base))
+pub fn factory() -> DynFilterFactory {
+    Box::from(DebugFactory::new())
 }
 
 impl Debug {
     /// Constructor for the Debug. Pass in a "id" to append a string to your log messages from this
     /// Filter.
-    fn new(base: &Logger, id: Option<String>) -> Self {
-        let log = match id {
-            None => base.new(o!("source" => "extensions::Debug")),
-            Some(id) => base.new(o!("source" => "extensions::Debug", "id" => id)),
-        };
 
-        Debug { log }
+    fn new(_: Option<String>) -> Self {
+        Debug {}
     }
 }
 
 impl Filter for Debug {
+    #[instrument]
```

Comment:
I would say better not to use this instrument attribute here!

it creates and enters a span every time the function is called and the parameters passed to the function are recorded as fields in the logs automatically.

if the number of parameters is high, our logs won’t be easily readable anymore!
with the result of this code here, we would have a log similar to:

`INFO caller_func_name:read{self=Debug ctx=ReadContext { endpoints: UpstreamEndpoints { endpoints: Endpoints([Endpoint { address: 127.0.0.1:80, metadata: MetadataView { known: Metadata { tokens: {} }, unknown: Mapping { map: {} } } }]), subset: None }, from: 127.0.0.1:90, contents: [104, 101, 108, 108, 111], metadata: {} }}: quilkin::filters::debug: Read filter event contents="hello"`

but if we try to just enter a span [like here](https://github.com/googleforgames/quilkin/pull/385/files#diff-fd5ade0f959fc048f50224e1dd2c16f8081fefbc5dde1f8f76f81b4909966805R63), the result can be:
`quilkin::filters::debug: Read filter event contents="hello"`

(as a json though)

Reply:
Makes sense, I know you can configure `instrument` to skip some stuff, so that might be worth looking into, but it's fine if you just enter a span manually.

Reply:
was thinking about having the same naming for our logs with that manual span, which that also is configurable in instrument like `#[instrument(name = “extensions::Compress::read”)]`

after conclusion on this, I’ll change other funcs to have them alike!